Source/driver/Castro.H

```diff
@@ -628,33 +595,33 @@ public:
 /// Compute initial ``dt``.
 ///
 /// @param finest_level     Index of finest level
-/// @param sub_cycle
+/// @param subcycle         unused
 /// @param n_cycle          the number of subcycles at this level
-/// @param ref_ratio        Vector of refinement ratios between levels
+/// @param ref_ratio        unused
 /// @param dt_level         Real Vector, will contain initial timestep at each level.
 /// @param stop_time        End time of simulation
 ///
     void computeInitialDt (int                   finest_level,
-                           int                   sub_cycle,
+                           int                   /*subcycle*/,
                            amrex::Vector<int>&           n_cycle,
-                           const amrex::Vector<amrex::IntVect>& ref_ratio,
+                           const amrex::Vector<amrex::IntVect>& /*ref_ratio*/,
                            amrex::Vector<amrex::Real>&          dt_level,
                            amrex::Real                  stop_time) override;
 
 ///
 /// Compute new ``dt``.
 ///
 /// @param finest_level     Index of finest level
-/// @param sub_cycle
+/// @param subcycle         unused
 /// @param n_cycle          the number of subcycles at this level
-/// @param ref_ratio        Vector of refinement ratios between levels
+/// @param ref_ratio        unused
 /// @param dt_min
 /// @param dt_level
 /// @param stop_time        End time of simulation
 /// @param post_regrid_flag Have we done regridding yet?
 ///
     void computeNewDt (int                   finest_level,
-                       int                   sub_cycle,
+                       int                   /*subcycle*/,
                        amrex::Vector<int>&           n_cycle,
                        const amrex::Vector<amrex::IntVect>& ref_ratio,
```

Comment:
```suggestion
                       const amrex::Vector<amrex::IntVect>& /*ref_ratio*/,
```
The docstring says that this is unused

Reply:
done